### PR TITLE
[16.0][IMP] account_statement_base: link to account move lines from statements

### PR DESCRIPTION
**Pull request by sergiobstoj**
Link to Account Move Lines in Statements

### CODE REVIEW

File: account_statement_base/tests/test_account_statement_base.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+from odoo.addons.account.tests.common import TestAccountReconciliationCommon
+
+
+@tagged("post_install", "-at_install")
+class TestReconciliationWidget(TestAccountReconciliationCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
```
Comment (aritzolea) [inline account_statement_base/tests/test_account_statement_base.py]: Avoid these line breaks

File: account_statement_base/tests/test_account_statement_base.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from odoo.addons.account.tests.common import TestAccountReconciliationCommon
+
+
+@tagged("post_install", "-at_install")
+class TestReconciliationWidget(TestAccountReconciliationCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.acc_bank_stmt_model = cls.env["account.bank.statement"]
```
Comment (aritzolea) [inline account_statement_base/tests/test_account_statement_base.py:12]: I would add a test in which you create a bank statement and then ensure you get it's move lines on the action's domain

File: account_statement_base/tests/test_account_statement_base.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        cls.current_assets_account = (
+            cls.env["account.account"]
+            .search(
+                [
+                    ("account_type", "=", "asset_current"),
+                    ("company_id", "=", cls.company.id),
+                ],
+                limit=1,
+            )
+            .copy()
```
Comment (aritzolea) [inline account_statement_base/tests/test_account_statement_base.py]: I wouldn't copy unless it is necessary.

**Review by aritzolea — APPROVED** [review]

**Comment by sergiobstoj** [comment]
@alexis-via looks good to you for merge?

**Comment by sergiobstoj** [comment]
@alexis-via is this OK for merge?

**Review by pedrobaeza — COMMENTED** [review]
Reduce the tests to one, as there's some duplication. Also remove trailing commas or use `Command.` for avoiding a lot of lines after the formater.

**Comment by pedrobaeza** [comment]
Please solve the conflicts and squash both commits.

**Review by pedrobaeza — APPROVED** [review]
Please fw-port it to 17

/ocabot merge minor

**Pull request merged**